tests/envs/test_envs.py

```diff
@@ -152,8 +156,7 @@ def check_rendered(rendered_frame, mode: str):
 render_mode_env_specs = [
     spec
     for spec in all_testing_env_specs
-    if ("mujoco" not in spec.entry_point or "v4" in spec.id)
-    and ("GymEnvironment" not in spec.entry_point)
+    if "mujoco" not in spec.entry_point or "v4" in spec.id
```

Comment:
I previously needed this to make sure that no Atari environment wrapped by `GymEnvironment` was being tested. In this PR I removed the Atari envs from `all_testing_env_specs` so this is no longer needed.